### DISCUSSION TITLE
Connecting Catalyst to Makeswift

### DISCUSSION THREAD

**Discussion by waynestratbucker** [bigcommerce/catalyst#726] [Q&A] [answered]
We are looking to get makeswift and catalyst to fully connect to one another. We are trying to find documentation on this, but not having any luck. 

Any feedback on this?

**Comment by chanceaclark** [comment, marked as answer]
Hey @waynestratbucker, there should be some documentation on Makeswifts website: https://docs.makeswift.com/integrations/bigcommerce

**Reply by sundarsan** [reply]
but is this makeswift catalyst code is updated with all the developments happening in catalyst Github Repo ?

**Reply by bookernath** [reply]
This is the branch to follow going forward: https://github.com/bigcommerce/catalyst/tree/integrations/makeswift
